docs/howto/deployment/wsgi/modwsgi.txt

```diff
@@ -48,9 +48,9 @@ your project package (``mysite`` in this example). This tells Apache to serve
 any request below the given URL using the WSGI application defined in that
 file.
 
-If you install your project's Python dependencies inside a `virtualenv`_, add
-the path to the virtualenv using ``WSGIPythonHome``. See the `mod_wsgi
-virtualenv guide`_ for more details.
+If you install your project's Python dependencies inside a :mod:`virtual
+environment <venv>`, add the path using ``WSGIPythonHome``. See the `mod_wsgi
```

Comment:
What do you see as the downside?

It provides more depth and information about the subject matter being discussed. It seems pretty routine that when Python features are mentioned in docs, they're also linked to.

Reply:
I guess it's that this links to a virtual environment tool, rather than docs about virtual environments more generally. There's this link which _may_ be better, but not sure how easy it is to link to using intersphinx? https://docs.python.org/3/library/venv.html#venv-def

Also - maybe I'm just being pedantic? 🤷

Reply:
Do you think it should link to https://docs.python.org/3/tutorial/venv.html instead?

Reply:
I think just leave this as suggested in the diff: maybe we could drop the link but at this moment in the reader's journey it's a nice, _Hey, you did remember to use a `venv` right?_ — repetition being to the key to recall.